### PR TITLE
Chapter 10: SEO content

### PR DESCRIPTION
**Pull request by ymschaap**
Result from #163 

The article the authors created for the SEO Chapter.

Reviewers did a round of feedback.

It could use another (very) critical look.

Don't merge yet.

### CODE REVIEW

**Comment by ymschaap** [comment]
Most of the comments I've 'resolved' now. Open to more suggestions (or un-resolves if you think that I overlooked something).

Overall I think is is already a pretty good result!

**Comment by rviscomi** [comment]
Thanks @ymschaap! Waiting on approval from the reviewers, then we can merge 🎉 

Reviewers, please give this a last look -- @rachellcostello @AVGP @clarkeclark @andylimn @voltek62 @AymenLoukil @catalinred

File: src/content/en/2019/seo.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+part_number: I
+chapter_number: 10
+title: SEO
+description: State of SEO 2019
+hero_image: ''
+authors: [ymschaap, rachellcostello, AVGP]
+reviewers: [clarkeclark, andylimn, voltek62]
```
Comment (AymenLoukil) [inline src/content/en/2019/seo.md]: To update with the current list of reviews may be

Comment (rviscomi) [inline src/content/en/2019/seo.md]:
```suggestion
reviewers: [clarkeclark, andylimn, voltek62, AymenLoukil, catalinred]
```

**Comment by rviscomi** [comment]
Thanks everyone! Merged.

**Pull request merged**